src/update_sdk.cmd

```diff
@@ -0,0 +1,43 @@
+REM This script replaces the SDK folder with the latest bits from the Azure SDK repository.
+REM It removes some files we do not need.  It currently targets the develop branch until fixes are
+REM merged into master.
+
```

Comment:
@stefangordon any updates on this? I'm still concerned about how we sync the C SDK, there's no way to track which version of the SDK is used. Previously, my bash script store the git commit hash as a text file.

Using a git submodule would be great.